docs/mkdocs.yml

```diff
@@ -36,25 +36,28 @@ nav:
   - Quick Start: direct/get-started.md
   - User Guide:
       - Overview: direct/user-guide-intro.md
-      - Prerequisites: direct/pre-reqs.md
-      - Acquire cluster for KubeFlex Hosting: direct/acquire-hosting-cluster.md
-      - Initialize KubeFlex Hosting cluster: direct/init-hosting-cluster.md
-      - Inventory and Transport Spaces: direct/its.md
-      - Workload Description Spaces: direct/wds.md
-      - Workload Execution Clusters: direct/wec.md
-      - Register a Workload Execution Cluster: direct/wec-registration.md
-      - Control:
-          - Overview: direct/control.md
-          - API reference (new tab): https://pkg.go.dev/github.com/kubestellar/kubestellar/api/control/v1alpha1
-          - Binding: direct/binding.md
-          - Transforming desired state: direct/transforming.md
-          - Combining reported state: direct/combined-status.md
-      - Core Helm chart: direct/core-chart.md
-      - Quick Start: direct/get-started.md
-      - Example Scenarios: direct/example-scenarios.md
-      - Third-party integrations:
-          - ArgoCD to WDS: direct/argo-to-wds1.md
-      - Best practices: direct/best-practices.md
+      - Quickstart: direct/get-started.md
+      - Setup:
+          - Prerequisites: direct/pre-reqs.md
+          - Acquire cluster for KubeFlex Hosting: direct/acquire-hosting-cluster.md
+          - Initialize KubeFlex Hosting cluster: direct/init-hosting-cluster.md
+          - Inventory and Transport Spaces: direct/its.md
+          - Workload Description Spaces: direct/wds.md
+          - Workload Execution Clusters: direct/wec.md
+          - Register a Workload Execution Cluster: direct/wec-registration.md
```

Comment:
Why do we need different section for WEC and registering ?

Reply:
Because creating a WEC is very different from registering one in an ITS. They have different dependencies: only the registration step has dependencies on the software prerequisites that we discuss. Think of edge computing scenarios: creating the edge cluster is distinct from registration; it is done by different parties at different times. Remember FIDO Device Onboard (FDO), a complicated architecture for the relationship between the manufacturer of the edge thingy, the supply chain between mfgr and customer, and the customer that integrates the edge thingy into their system.